src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -1,11 +1,10 @@
 package cromwell.engine.backend.jes
 
-import java.io.File
 import java.nio.file.Paths
 
 import cromwell.binding._
 import cromwell.binding.values.WdlFile
-import cromwell.engine.backend.jes.JesBackend.{JesInput, JesOutput}
+import cromwell.engine.backend.jes.JesBackend.{JesOutput, localFilePathFromRelativePath}
```

Comment:
Opinions i've seen vary as wildly as they do in java-land, which is to say there are as many opinions on imports as there are users